### ISSUE TITLE
Example download issue 

### ISSUE THREAD

**Issue by YubinXie** [broadinstitute/CellBender#95]
Hi I am trying to run the example code and got the following issue when downloading the tiny example:

```
(pytorch) bash-4.2$ python generate_tiny_10x_pbmc.py 
Downloading pbmc4k (CellRanger 2.1.0, v2 Chemistry)...
Could not retrieve pbmc4k (CellRanger 2.1.0, v2 Chemistry) -- cannot continue!
```

**Comment by sjfleming** [comment]
Thanks for pointing this out!  I will check and see if the CellRanger website has changed the location of its data files.

**Comment by sjfleming** [comment]
Hmm, I still seem to be able to download the file here:
http://cf.10xgenomics.com/samples/cell-exp/2.1.0/pbmc4k/pbmc4k_raw_gene_bc_matrices.tar.gz

I guess it could either be a permissions issue (if you don't have permission to write to the current directory where you're running this script), or could it be an internet connectivity issue?

**Comment by sjfleming** [comment]
The relevant line of code is here
https://github.com/broadinstitute/CellBender/blob/78f005fc8dc9313328d4ef7a6f4ed985fa253187/examples/remove_background/generate_tiny_10x_pbmc.py#L25

**Comment by joshuahwu** [comment]
I had the same issue with the example tutorial. Ended up manually downloading from the link and commenting out the relevant line of code. This worked.

**Comment by sjfleming** [comment]
Thanks @joshuahwu I will try to debug this more and see why it's failing some of the time...

**Comment by julie-jch** [comment]
Was there any solution to this?
I'm running and getting the same error


$ python generate_tiny_10x_pbmc.py
Downloading pbmc4k (CellRanger 2.1.0, v2 Chemistry)...
Could not retrieve pbmc4k (CellRanger 2.1.0, v2 Chemistry) -- cannot continue!

**Comment by RussBainer** [comment]
I ran into this too. Running from console I get a 403, perhaps the request needs a header?

**Comment by sjfleming** [comment]
Closed by #238
